site/tutorials/gettings_started.md

```diff
@@ -0,0 +1,293 @@
+
+<div class="vl-example">
+{
+  "data": {
+    "values": [
+      {"a": "C", "b": 2}, {"a": "C", "b": 7}, {"a": "C", "b": 4},
+      {"a": "D", "b": 1}, {"a": "D", "b": 2}, {"a": "D", "b": 6},
+      {"a": "E", "b": 8}, {"a": "E", "b": 4}, {"a": "E", "b": 7}
+    ]
+  },
+  "mark": "point",
+  "encoding": {
+    "x": {"field": "a", "type": "ordinal"},
+    "y": {"field": "b", "type": "quantitative"}
+  }
+}
+</div>
+
+Now we can see the raw data points. Note that Vega-Lite automatically added grid lines to the y-axis to help you see the magnitude of the `b` values.
+
+## Aggregate Data
+
+Vega-Lite can do more than just visualizing raw data. By adding `"aggregate": "mean"` to the definition of the y encoding channel, we can see the average value in each category. For example, the average value of category `D` is `(1 + 2 + 6)/3 = 9/3 = 3`.
+
+<div class="vl-example">
+{
+  "data": {
+    "values": [
+      {"a": "C", "b": 2}, {"a": "C", "b": 7}, {"a": "C", "b": 4},
+      {"a": "D", "b": 1}, {"a": "D", "b": 2}, {"a": "D", "b": 6},
+      {"a": "E", "b": 8}, {"a": "E", "b": 4}, {"a": "E", "b": 7}
+    ]
+  },
+  "mark": "point",
+  "encoding": {
+    "x": {"field": "a", "type": "ordinal"},
+    "y": {"field": "b", "type": "quantitative", "aggregate": "mean"}
+  }
+}
+</div>
+
+Great! You computed the aggregate values for each category and visualized the resulting value as a point. Typically aggregated values for categories are visualized using bar charts. To create a bar chart, we have to change the mark type from `point` to `bar`.
+
+
+```diff
+- "mark": "point"
++ "mark": "bar"
+```
+
+<div class="vl-example">
+{
+  "data": {
+    "values": [
+      {"a": "C", "b": 2}, {"a": "C", "b": 7}, {"a": "C", "b": 4},
+      {"a": "D", "b": 1}, {"a": "D", "b": 2}, {"a": "D", "b": 6},
+      {"a": "E", "b": 8}, {"a": "E", "b": 4}, {"a": "E", "b": 7}
+    ]
+  },
+  "mark": "bar",
+  "encoding": {
+    "x": {"field": "a", "type": "ordinal"},
+    "y": {"field": "b", "type": "quantitative", "aggregate": "mean"}
+  }
+}
+</div>
+
+Note that you don't have to specify that you want a vertical bar chart. In fact, we can change the visualization to a horizontal bar chart by swapping the `x` and `y` channels.
+
+<div class="vl-example">
+{
+  "data": {
+    "values": [
+      {"a": "C", "b": 2}, {"a": "C", "b": 7}, {"a": "C", "b": 4},
+      {"a": "D", "b": 1}, {"a": "D", "b": 2}, {"a": "D", "b": 6},
+      {"a": "E", "b": 8}, {"a": "E", "b": 4}, {"a": "E", "b": 7}
+    ]
+  },
+  "mark": "bar",
+  "encoding": {
+    "y": {"field": "a", "type": "ordinal"},
+    "x": {"field": "b", "type": "quantitative", "aggregate": "mean"}
+  }
+}
+</div>
+
+## Customize your Visualization
+
+Vega-Lite automatically provides a large number of defaults, which you can override. For example, to set change the title of the x-axis from `MEAN(b)` to `average of b`, we can set the title property of the axis in the `x` channel.
```

Comment:
Vega-Lite automatically provides sensible default properties for the visualization.  
However, you can further customize these values.  For example, ...

(Large number isn't the point. -- actually that sounds complicated.)